src/redis_cache.rs

```diff
@@ -107,11 +125,11 @@ impl RedisCacheStore {
   }
 
   fn cache_key<S: Display>(&self, key: S) -> String {
-    format!("cache:{}:{}", self.ns, key)
+    format!("{}:{}:{}", CACHE_PREFIX, self.ns, key)
```

Comment:
I just realized a namespace isn't the key prefix -- I didn't follow that convention in [#26](https://github.com/superfly/fly.rs/pull/26/files#diff-fd66be3cc72412676e7ddeb256ef6c58R25). What's the reason for the namespace being inside the key rather than a prefix?

Reply:
I'm not sure why it was decided like that. We could change it. I was following what we had in fly-node. I did that a bit blindly for some things.

I think it probably depends how we want to be able to mass-query our redis cache. We've never had to do it yet though.

For some reason, my brain is happier with `cache:1:name` than `1:cache:name`, but that's random heh.

Reply:
I've always done the namespace first because that's what the redis-namespace gem did back in the day and I got used to it, but it doesn't really matter. It would be cool to create a little helper for the redis code to use everywhere though.